migrations/scripts/2_deploy_board.js

```diff
@@ -0,0 +1,104 @@
+const realm = process.env.WITNET_EVM_REALM || "default"
+const addresses = require("../addresses")[realm]
+const settings = require("../settings")
+
+let WitnetProxy, WitnetRequestBoard
+
+module.exports = async function (deployer, network, accounts) {
+  try {
+    WitnetProxy = artifacts.require(settings.artifacts[realm].WitnetProxy || settings.artifacts.default.WitnetProxy)
```

Comment:
As `realm` is already `x || 'default`, the right hand side of this expression is unreachable.

```suggestion
    WitnetProxy = artifacts.require(settings.artifacts[realm].WitnetProxy)
```

Reply:
Mmm, 'realm` could be set to `omgx`, but still there could be no specific implementation name specified for the `WitnetProxy` artifact. In this case, `settings.artifacts[realm].WitnetProxy === null`, but still, we want use the default implementation name set at `settings.default.WitnetProxy` (i.e. "WitnetProxy").